Adaptors/MongoDB/src/PartitionTable.cs

```diff
@@ -0,0 +1,146 @@
+//   W. Kirschenmann   <wkirschenmann@aneo.fr>
+//   J. Gurhem         <jgurhem@aneo.fr>
+//   D. Dubuc          <ddubuc@aneo.fr>
+//   L. Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F. Lemaitre       <flemaitre@aneo.fr>
+//   S. Djebbar        <sdjebbar@aneo.fr>
+//   J. Fonseca        <jfonseca@aneo.fr>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using ArmoniK.Core.Adapters.MongoDB.Common;
+using ArmoniK.Core.Adapters.MongoDB.Table.DataModel;
+using ArmoniK.Core.Common;
+using ArmoniK.Core.Common.Exceptions;
+using ArmoniK.Core.Common.Storage;
+
+using Microsoft.Extensions.Logging;
+
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+
+namespace ArmoniK.Core.Adapters.MongoDB;
+
+public class PartitionTable : IPartitionTable
+{
+  private readonly ActivitySource                                                    activitySource_;
+  private readonly ILogger<TaskTable>                                                logger_;
+  private readonly MongoCollectionProvider<PartitionData, PartitionDataModelMapping> partitionCollectionProvider_;
+  private readonly SessionProvider                                                   sessionProvider_;
+
+  private bool isInitialized_;
+
+  public PartitionTable(SessionProvider                                                   sessionProvider,
+                        MongoCollectionProvider<PartitionData, PartitionDataModelMapping> partitionCollectionProvider,
+                        ILogger<TaskTable>                                                logger,
+                        ActivitySource                                                    activitySource)
+  {
+    sessionProvider_             = sessionProvider;
+    partitionCollectionProvider_ = partitionCollectionProvider;
+    logger_                      = logger;
+    activitySource_              = activitySource;
+  }
+
+  public ValueTask<bool> Check(HealthCheckTag tag)
+    => ValueTask.FromResult(isInitialized_);
+
+  public Task Init(CancellationToken cancellationToken)
+  {
+    if (!isInitialized_)
+    {
+      sessionProvider_.Get();
+      partitionCollectionProvider_.Get();
+      isInitialized_ = true;
+    }
+
+    return Task.CompletedTask;
+  }
+
```

Comment:
`  /// <inheritdoc />` missing in all functions

Reply:
This is not necessary as VS is perfectly able to find the doc without it. As for docfx, the behavior of `/// <inheritdoc />` does not seem to add the doc of the interface in the implementation classes.

Reply:
Oks